### ISSUE TITLE
display of grid question

### ISSUE THREAD

**Issue by rdewijn** [tercen/pamsoft_grid_shiny_operator#2]
As far as I can see the displayed grid in this gridviewer is a grid obtained after grid localization AND segmentation. Correct?
For the grid displayed after using a "New Grid" (re-grid) this appears to be not the case (?)

**Comment by tsmonteiro** [comment]
That is correct. Segmentation will then be run on the quantitation step afterwards (only for the newly created grids)

**Comment by rdewijn** [comment]
No way to view the final regridded grid?

**Comment by tsmonteiro** [comment]
Within the shiny app, I don't think so.
